### DISCUSSION TITLE
Adding Mailable To From Subject

### DISCUSSION THREAD

**Discussion by dillingham** [spatie/ray#77] [Ideas]
Sending emails to log and automatically seeing in Ray is a game changer.

Would be nice if there was "To: ... From: ... Subject:...".

I would prefer below the rendered mailable personally.

**Comment by freekmurze** [comment]
That's a good idea, but it would require changes on the Laravel side, they don't write that info now to the log.
If they would write it to the log, I would probably be able to show that info in Ray.

**Reply by dillingham** [reply]
Interesting.. Im seeing it when getting email verification from Jetstream. 

Using `MAIL_MAILER=log`

```
[2021-01-09 10:22:45] local.DEBUG: Message-ID: <024c945eef9710a34fc7b2a7e0184a63@swift.generated>
Date: Sat, 09 Jan 2021 10:22:45 +0000
Subject: Verify Email Address
From: 
To: mizewolu@mailinator.com
MIME-Version: 1.0
Content-Type: multipart/alternative;
 boundary="_=_swift_1610187765_46478a526aea4be0d5bbcc588046514d_=_"

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>

....
```

**Comment by freekmurze** [comment]
O, I didn't see that Subject, From etc... in any of my tests. With this info we'll probably be able to get that info 🎉

**Reply by freekmurze** [reply]
If you need this quick, you could take a stab at supporting this. The relevant code is here: https://github.com/spatie/laravel-ray/blob/dc4ef1719a2c66e1bc4c8a6c4694412c48c49a2c/src/RayServiceProvider.php#L98-L105

When a mail is detected, with that extra info, you could send a `MailablePayload`.

If you don't have the time to add this, or it's a bit too difficult, I'll spend some time in the near feature adding this myself.
